worlds/cvcotm/locations.py

```diff
@@ -0,0 +1,265 @@
+    lname.ug13:  CVCotMLocationData(0x88, 0xD5AB8,  9),
+    lname.ug15:  CVCotMLocationData(0x89, 0xD5BD8,  9),
+    lname.ug20:  CVCotMLocationData(0xAC, 0xD5470,  9, type="boss"),
+    # Underground Warehouse
+    lname.uw1:   CVCotMLocationData(0x75, 0xD48DC,  8),
+    lname.uw6:   CVCotMLocationData(0x76, 0xD4D20,  8),
+    lname.uw8:   CVCotMLocationData(0x77, 0xD4BA0,  8),
+    lname.uw9:   CVCotMLocationData(0x7E, 0xD53EC,  8),
+    lname.uw10:  CVCotMLocationData(0x78, 0xD4C84,  8),
+    lname.uw11:  CVCotMLocationData(0x79, 0xD4EC4,  8),
+    lname.uw14:  CVCotMLocationData(0x7F, 0xD5410,  8),
+    lname.uw16:  CVCotMLocationData(0x7A, 0xD5050,  8),
+    lname.uw16b: CVCotMLocationData(0x7B, 0xD505C,  8),
+    lname.uw19:  CVCotMLocationData(0x7C, 0xD5344,  8),
+    lname.uw23:  CVCotMLocationData(0xAE, 0xD53B0,  8, type="boss"),
+    lname.uw24:  CVCotMLocationData(0x80, 0xD5434,  8),
+    lname.uw25:  CVCotMLocationData(0x7D, 0xD4FC0,  8),
+    # Underground Waterway
+    lname.uy1:   CVCotMLocationData(0x93, 0xD5F98, 10),
+    lname.uy3:   CVCotMLocationData(0x8B, 0xD5FEC, 10),
+    lname.uy3b:  CVCotMLocationData(0x8A, 0xD5FE0, 10),
+    lname.uy4:   CVCotMLocationData(0x94, 0xD697C, 10),
+    lname.uy5:   CVCotMLocationData(0x8C, 0xD6214, 10),
+    lname.uy7:   CVCotMLocationData(0x8D, 0xD65A4, 10),
+    lname.uy8:   CVCotMLocationData(0x95, 0xD69A0, 10),
+    lname.uy9:   CVCotMLocationData(0x8E, 0xD640C, 10),
+    lname.uy9b:  CVCotMLocationData(0x8F, 0xD6418, 10),
+    lname.uy12:  CVCotMLocationData(0x90, 0xD6730, 10),
+    lname.uy12b: CVCotMLocationData(0x91, 0xD673C, 10),
+    lname.uy13:  CVCotMLocationData(0x92, 0xD685C, 10),
+    lname.uy17:  CVCotMLocationData(0xAF, 0xD6940, 10, type="boss"),
+    lname.uy18:  CVCotMLocationData(0x96, 0xD69C4, 10),
+    # Observation Tower
+    lname.ot1:   CVCotMLocationData(0x9D, 0xD6B38, 11),
+    lname.ot2:   CVCotMLocationData(0xA4, 0xD760C, 12),
+    lname.ot3:   CVCotMLocationData(0x9F, 0xD72E8, 12),
+    lname.ot5:   CVCotMLocationData(0xA5, 0xD75E8, 12),
+    lname.ot8:   CVCotMLocationData(0xA0, 0xD71EC, 12),
+    lname.ot9:   CVCotMLocationData(0xA2, 0xD6FE8, 12),
+    lname.ot12:  CVCotMLocationData(0xA6, 0xD75C4, 12),
+    lname.ot13:  CVCotMLocationData(0xA3, 0xD6F64, 12),
+    lname.ot16:  CVCotMLocationData(0xA1, 0xD751C, 12),
+    lname.ot20:  CVCotMLocationData(0xB0, 0xD6E20, 12, type="boss"),
+    # Ceremonial Room
+    lname.cr1:   CVCotMLocationData(0xA7, 0xD7690, 13),
+    lname.dracula: CVCotMLocationData(iname.dracula, None, None),
+    # Battle Arena
+    lname.ba24:  CVCotMLocationData(0xB2, 0xD7D20, 14, type="arena"),
+    lname.arena_victory:  CVCotMLocationData(iname.shinning_armor, None, None),
+ }
+
+
+def get_location_names_to_ids() -> Dict[str, int]:
+    return {name: cvcotm_location_info[name].code+BASE_ID for name in cvcotm_location_info
+            if isinstance(cvcotm_location_info[name].code, int)}
```

Comment:
`is not None` is definitely preferrable over `isinstance(..., int)`, it's not a huge deal tho.

*low priority*

Reply:
The code attribute is either an int or a string, actually. When creating the locations themselves, if code is a string, the ID will become None and the string is the name of the item to put locked on that location.

Reply:
Oh 🤦 my bad